common/upgets/api_resource_spot_finder.lua

```diff
@@ -324,6 +324,7 @@ local function GetSpotsMetal()
 	end
 
 	-- Final processing
+	local uDefID = UnitDefNames["armmex"].id
```

Comment:
This is hardcoding the unit def ID of `armmex`, what about all the other metal extractor units?

Reply:
> Downside: [GetBuildingPositions](https://github.com/beyond-all-reason/Beyond-All-Reason/blob/master/common/upgets/api_resource_spot_finder.lua#L152) has loops and requires UnitDef.

Reply:
If the value is map-defined, why not get it directly from the map then?